app/javascript/src/shared/styles/fontfaces.js

```diff
@@ -0,0 +1,56 @@
+// @flow
+import { css } from 'glamor'
+import robotoLightWOFF2 from '../../../fonts/roboto-light.woff2'
+import robotoLightWOFF from '../../../fonts/roboto-light.woff'
+import robotoLightTTF from '../../../fonts/roboto-light.ttf'
+
+import robotoRegularWOFF2 from '../../../fonts/roboto-regular.woff2'
+import robotoRegularWOFF from '../../../fonts/roboto-regular.woff'
+import robotoRegularTTF from '../../../fonts/roboto-regular.ttf'
+
+import robotoBoldWOFF2 from '../../../fonts/roboto-bold.woff2'
+import robotoBoldWOFF from '../../../fonts/roboto-bold.woff'
+import robotoBoldTTF from '../../../fonts/roboto-bold.ttf'
+
+import zillaSlabRegularWOFF2 from '../../../fonts/zillaslab-regular.woff2'
+import zillaSlabRegularWOFF from '../../../fonts/zillaslab-regular.woff'
+import zillaSlabRegularTTF from '../../../fonts/zillaslab-regular.ttf'
```

Comment:
i don't think we need ttf if we have woff/2, do you think we can get rid of it?

Reply:
Sure